docs/Clearwater_stress_testing.md

```diff
@@ -12,82 +12,78 @@ package installed.
 
 ### What they do
 
-Clearwater SIP stress nodes run a standard SIPp script against your bono cluster.  The bono nodes translate this into traffic for sprout and this generates traffic on homestead and homer.  The nodes log their success/failure to `/var/log/clearwater-sip-stress` and also restart SIPp (after a 30s cool-down) if anything goes wrong.
+Clearwater SIP stress nodes provide a set of SIPp scripts to run against your Sprout cluster. From Sprout's point of view, they look like a P-CSCF.  The nodes log their success/failure to `/var/log/clearwater-sip-stress` and also print a human-readable summary of the stress (with informatin about percentage of failed calls, average latency, and so on).
 
-The SIPp stress starts as soon as the node comes up - this is by design, to avoid you having to manually start 50+ nodes generating traffic.  If you want to stop them from generating traffic, turn the whole node down.
+They follow a fixed load profile, assuming 1.3 calls/hour for each subscriber (split equally between incoming and outgoing calls) and 2 re-registrations per hour.
 
-Each SIP stress node picks a single bono to generate traffic against.  This bono is chosen by matching the bono node's index against the SIP stress node's index.  As a result, it's important to always give SIP stress nodes an index.
+## Deploying a stress node
 
-### How they work
+Follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision the number of subscribers you want. As a general guideline, we'd expect a small deployment (with one Sprout, Homestead and optionally a Ralf node, each with one full CPU core) to handle at least 50,000 subscribers.
 
-The clearwater-sip-stress package includes two important scripts.
+You should also ensure that the `reg_max_expires` setting is set to 1800 rather than the default of 300 - see our [configuration guide](Clearwater_Configuration_Options_Reference.md) for instructions. This matches the assumptions in our load profile of 2 re-registers per hour.
```

Comment:
I'm happy for this to be the new default - worth sending an PSA to the team on this though